lib/index.js

```diff
@@ -86,22 +87,27 @@ Optimizely.prototype.initialize = function() {
  */
 
 Optimizely.prototype.track = function(track) {
+  var opts = this.options;
+  var eventProperties = track.properties();
   // Use the new-style API (which is compatible with Classic and X)
   var payload = {
     type: 'event',
     eventName: track.event(),
-    tags: track.properties()
+    tags: eventProperties
   };
 
-  if (payload.tags.revenue) {
+  if (eventProperties.revenue && opts.sendRevenueOnlyForOrderCompleted) {
+    if (track.event() === 'Order Completed') {
+      payload.tags.revenue = Math.round(payload.tags.revenue * 100);
```

Comment:
I believe that this should be eventProperties.revenue=Math.round(payload.tags.revenue * 100); here and on 104 below so that when eventProperties is sent to full stack revenue is also formatted there.